app/views/items/_item.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<% current_user.items.each do |item| %>
```

Comment:
Take the loop out of this - we only put the code to display one item in the partial.